lms/templates/static_templates/honor.html

```diff
@@ -0,0 +1,34 @@
+<%page expression_filter="h"/>
+<%! from django.utils.translation import ugettext as _ %>
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("Honor Code")}</%block>
+
+<main id="main" aria-label="Content" tabindex="-1">
```

Comment:
Should add an <h1>Honor Code</h1>, like the [Terms of Service page](https://github.com/hks-epod/epodx-theme/blob/master/lms/templates/static_templates/tos.html#L8) does.